frontend/components/software/edit/information/ValidateConceptDoi.tsx

```diff
@@ -0,0 +1,56 @@
+// SPDX-FileCopyrightText: 2022 Dusan Mijatovic (dv4all)
+// SPDX-FileCopyrightText: 2022 dv4all
+//
+// SPDX-License-Identifier: Apache-2.0
+
+import {useState} from 'react'
+import Button from '@mui/material/Button'
+import CircularProgress from '@mui/material/CircularProgress'
+import FactCheckIcon from '@mui/icons-material/FactCheck'
+
+import {softwareInformation as config} from '../editSoftwareConfig'
+import useSnackbar from '~/components/snackbar/useSnackbar'
+import {getSoftwareVersionInfoForDoi} from '~/utils/getDataCite'
+
+export default function ValidateConceptDoi({doi}: { doi: string }) {
+  const {showErrorMessage,showSuccessMessage, showInfoMessage} = useSnackbar()
+  const [loading,setLoading]=useState(false)
+
+  async function validateDoi() {
+    setLoading(true)
+    const info = await getSoftwareVersionInfoForDoi(doi)
+    if (info) {
+      if (info.versionOfCount === 0) {
+        showSuccessMessage(`Valid concept DOI: ${doi}`)
```

Comment:
Minor rephrase suggestion:

```suggestion
        showSuccessMessage(`The DOI ${doi} is a valid Concept DOI.`)
```